tool/jenkins/jenkins_build.sh

```diff
@@ -51,35 +51,46 @@ docker pull madlib/postgres_13:jenkins
 # Launch docker container with volume mounted from workdir
 echo "-------------------------------"
 cat <<EOF
-docker run -d -e POSTGRES_PASSWORD=postgres --name madlib -v "${workdir}":/madlib madlib/postgres_13:jenkins | tee logs/docker_setup.log
+docker run -d -t -e POSTGRES_PASSWORD=postgres --name madlib --ulimit core=-1  --privileged -v "${workdir}":/madlib madlib/postgres_13:jenkins | tee logs/docker_setup.log
 EOF
-docker run -d -e POSTGRES_PASSWORD=postgres --name madlib -v "${workdir}":/madlib madlib/postgres_13:jenkins | tee logs/docker_setup.log
+docker run -d -t -e POSTGRES_PASSWORD=postgres --name madlib --ulimit core=-1  --privileged -v "${workdir}":/madlib madlib/postgres_13:jenkins | tee logs/docker_setup.log
+# -c log_statement=all -c logging_collector=on -c log_directory=logs -c log_filename=postgresql.log
```

Comment:
we can probably remove this commented line

Reply:
They were useful while I was trying to collect postgres logs. I'll add a comment to mention that.